src/evergreen/api.py

```diff
@@ -829,7 +829,8 @@ def single_test_by_task_and_test_file(self, task_id: str, test_file: str) -> Tst
         :return: the test for the specified task.
         """
         url = self._create_url(f"/tasks/{task_id}/tests")
-        return Tst(self._call_api(url, params={"test_name": test_file}).json(), self)
+        test = self._call_api(url, params={"test_name": test_file}).json()
```

Comment:
We should avoid returning raw json. We will want to wrap the results in `Tst` objects.